calico-enterprise/multicluster/set-up-multi-cluster-management/helm-install/create-a-management-cluster-helm.mdx

```diff
@@ -47,7 +47,7 @@ If you are installing on a cluster installed by EKS, GKE, AKS or Mirantis Kubern
 1. If you are installing on a cluster installed by EKS, GKE, AKS or Mirantis Kubernetes Engine (MKE), set the `kubernetesProvider` as described in the [Installation reference](../../../reference/installation/api.mdx#operator.tigera.io/v1.Provider). For example:
 
   ```bash
-  echo '{ installation: {kubernetesProvider: EKS }}' > values.yaml
+  echo 'installation: {kubernetesProvider: EKS }' > values.yaml
```

Comment:
This needs to be changed for the managed cluster page as well.

nit: we could probably correct the spacing while we're at it, e.g.
`installation: { kubernetesProvider: EKS }`